testSetup.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const serializer = require('jest-serializer-path');
+
+expect.addSnapshotSerializer(serializer);
```

Comment:
Do you need to register this in both the config and a test setup file? I thought it was one or the other.

Reply:
I think I am only registering it once in the test setup file.